modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/StructuredContentResourceImpl.java

```diff
@@ -16,9 +16,6 @@
 
 import com.liferay.document.library.kernel.service.DLAppService;
```

Comment:
I think this is a good smell. A refactor that allow you to get rid of a lot of imports